addon-test-support/-private/patch-test-loader.js

```diff
@@ -36,9 +38,13 @@ export default function patchTestLoader(TestLoader) {
     testLoader._testModules = [];
     _super.loadModules.apply(testLoader, arguments);
 
-    const splitModules = splitTestModules(testLoader._testModules, split, partitions);
+    if (weighted) {
```

Comment:
I'm not sure if this will somehow cause an issue or not. The old code put the split list into 'splitModules', but left testLoader._testModules alone, so _testModules is the original 1-D array (isn't it?) The new code puts the 2-D array into _testModules. Does that have any effects elsewhere?